### PR TITLE
Add the functionality for additionalTrustBundles

### PR DESCRIPTION
**Pull request by yannickstruyf3**
**What this PR does / why we need it**:
- Upgrade prism-go to 0.3.2
- Add additional trustbundle functionality in CCM
- Fix CVE for /x/text

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
N/A

**How Has This Been Tested?**:

```
make unit-test-html
```

```
Ran 62 of 62 Specs in 0.413 seconds
SUCCESS! -- 62 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestCloudProviderNutanix (0.41s)
```

Also did manual tests with CCM.

**Special notes for your reviewer**:

**Release note**:
```release-note
- Upgrade prism-go to 0.3.2
- Add additional trustbundle functionality in CCM
- Fix CVE for /x/text
- Fix CVE for /x/net
```

### CODE REVIEW

**Review by tuxtof — APPROVED** [review]
LGTM

**Review by tuxtof — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -27,3 +31,41 @@ unit-test-html: unit-test
...
+## --------------------------------------
+
+.PHONY: deployment-manifests
+deployment-manifests:
+	mkdir -p $(ARTIFACTS)/manifests
+	cat manifests/*.yaml | envsubst > $(ARTIFACTS)/manifests/deploy_ccm.yaml
+
+.PHONY: deploy
+deploy: deployment-manifests
+	kubectl apply -f $(ARTIFACTS)/manifests/deploy_ccm.yaml
```
Comment (thunderboltsid) [inline Makefile:71]: /n

Comment (yannickstruyf3) [reply Makefile]: Modified

File: Makefile [hunk 2]
```diff
@@ -27,3 +31,41 @@ unit-test-html: unit-test
 ## --------------------------------------
 
 include ./openshift/openshift.mk
+
+## --------------------------------------
+## Create Image
+## --------------------------------------
+
+LOCALBIN ?= $(shell pwd)/bin
```
Comment (thunderboltsid) [inline Makefile:39]: shoudn't this use `REPO_ROOT`?

Comment (yannickstruyf3) [reply Makefile]: Good catch. Modified

**Comment by thunderboltsid** [comment]
lgtm

**Review by thunderboltsid — APPROVED** [review]

**Review by tuxtof — APPROVED** [review]

**Pull request merged**